solutions/instances/variables.tf

```diff
@@ -43,6 +43,188 @@ variable "prefix" {
+# IBM Cloud Logs
+##############################################################################
+variable "cloud_logs_provision" {
+  description = "Set it to true to provision an IBM Cloud Logs instance"
+  type        = bool
+  default     = false
+}
+
+variable "cloud_logs_plan" {
+  type        = string
+  description = "The IBM Cloud Logs plan to provision. Possible values: standard"
+  default     = "standard"
+}
+
+variable "cloud_logs_instance_name" {
+  type        = string
+  description = "The name of the IBM Cloud Logs instance to create. If a prefix input variable is passed, it is prefixed to the value in the `<prefix>-value` format."
+  default     = "cloud-logs"
+}
+
+variable "cloud_logs_region" {
+  description = "The region where Cloud Logs instances resources are created."
+  type        = string
+  default     = null
+}
+
+variable "cloud_logs_tags" {
+  type        = list(string)
+  description = "The resource tags that are associated with the IBM Cloud Logs instance (`Optional`, `array of strings`)."
+  default     = []
+}
+
+variable "cloud_logs_access_tags" {
+  type        = list(string)
+  description = "A list of access tags to apply to the Cloud Logs instance. Maximum length: 128 characters. Possible characters are A-Z, 0-9, spaces, underscores, hyphens, periods, and colons. [Learn more](https://cloud.ibm.com/docs/account?topic=account-access-tags-tutorial)."
+  default     = []
+
+  validation {
+    condition = alltrue([
+      for tag in var.cloud_logs_access_tags : can(regex("[\\w\\-_\\.]+:[\\w\\-_\\.]+", tag)) && length(tag) <= 128
+    ])
+    error_message = "Tags must match the regular expression \"[\\w\\-_\\.]+:[\\w\\-_\\.]+\". For more information, see https://cloud.ibm.com/docs/account?topic=account-tag&interface=ui#limits."
+  }
+}
+
+variable "cloud_logs_service_endpoints" {
+  description = "The type of endpoint for the IBM Cloud Logs instance. Possible values: `public`, `private`, `public-and-private`."
+  type        = string
+  default     = "public-and-private"
+  validation {
+    condition     = contains(["public", "private", "public-and-private"], var.cloud_logs_service_endpoints)
+    error_message = "The specified service_endpoints is not a valid selection"
+  }
+}
+
+variable "enable_cloud_logs_data" {
+  type        = bool
+  description = "Whether to enable cloud logs data to the Object Storage bucket."
+  default     = true
+}
+
+variable "enable_cloud_logs_metrics" {
+  type        = bool
+  description = "Whether to enable cloud logs metric data to the Object Storage bucket."
+  default     = true
+}
+
+variable "enable_en_cloud_logs_integration" {
+  type        = bool
+  description = "Whether to enable event notification integration for IBM Cloud Logs instance"
+  default     = false
+}
+
+variable "existing_en_instance_crn" {
+  type        = string
+  description = "The CRN of the existing event notification instance."
```

Comment:
Specify that when a value passing a value here, `enable_en_cloud_logs_integration` must be set to true to enable it